app/api/v2/commercial/accounts.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
```

Comment:
commercial is not correct, I think we would be in organizations scope.

Reply:
@calj Please check the following endpoint. Is this correct?

List

1. List accounts which user have access.
GET `/api/v2/barong/organization/accounts`

2. List all organizations 
GET `/api/v2/barong/organization/organizations`

3. Get organization info
GET `/api/v2/barong/organization?oid=...`

4. List organization users
GET `/api/v2/barong/organization/users?oid=...`

5. List organization subunits
GET `/api/v2/barong/organization/organizations?oid=...`

Create

1. Create organization
POST `/api/v2/barong/organization`

2. Create subunit in the organization
POST `/api/v2/barong/organization/account`

3. Add existing user into organization/subunit
POST `/api/v2/barong/organization/user`

Update

1. Update organization info
POST `/api/v2/barong/organization/update`

2. Update organization settings
POST `/api/v2/barong/organization/setting`

Delete

1. Delete subunit in the organization
DELETE `/api/v2/barong/organization/account`

2. Delete user in the organization
DELETE `/api/v2/barong/organization/user`